game/static/game/css/auth.css

```
@@ -12,15 +12,66 @@ body {
-    font-size: 2rem;
+/* .header { margin-bottom: 30px; text-align: center; } */
+/* ============================================================
+   Header
+   ============================================================ */
+
+   .header { 
+    padding: 20px 0 8px; 
+    display: flex;
+    flex-direction: column; /* Stack logo container and subtitle <p> */
+
+    align-items: center;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)